docs/configuration.md

```diff
@@ -1809,6 +1809,14 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.shuffle.maxChunksBeingTransferred</code></td>
+  <td>Long.MAX_VALUE</td>
+  <td>
+    The max number of chunks being transferred at the same time. This config helps avoid OOM on
```

Comment:
nit: `The max number of chunks allowed to being transferred at the same time on shuffle service.`